docs/data-sources/cloud_connection.md

```diff
@@ -0,0 +1,45 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "streamnative_cloud_connection Data Source - terraform-provider-streamnative"
+subcategory: ""
+description: |-
+  
+---
+
+# streamnative_cloud_connection (Data Source)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The service account name
```

Comment:
Please add `connection_name` for avoid docs error description

Reply:
Updated